docker/spark/scripts/entrypoint-submit.sh

```diff
@@ -5,6 +5,7 @@ SPARK_MASTER_PORT=${SPARK_MASTER_PORT:=7077}
 SPARK_WEBUI_PORT=${SPARK_WEBUI_PORT:=8080}
 SPARK_MASTER_HOST=${SPARK_MASTER_HOST:=spark-master}
 MAX_RETRIES=${MAX_RETRIES:=300}
+MAX_RETRIES_JOB=${MAX_RETRIES_JOB:=1}
```

Comment:
remove this entire code, restarts are the responsibility fo the runner (nomad, yarn, etc.)